src/DKG.sol

```diff
@@ -51,17 +64,18 @@ interface IDKG {
     /// @param from The address of the participant.
     /// @param index The index of the participant.
     /// @param tmpKey The temporary key of the participant.
-    event NewParticipant(address from, uint32 index, uint256 tmpKey);
+    event NewParticipant(address from, uint32 index, G1Point tmpKey);
 
     /// @notice Emitted when a deal is submitted during the deal phase.
-    /// @param dealerIdx The index of the dealer submitting the deal.
+    /// @param dealerIdx The address of the dealer submitting the deal.
     /// @param bundle The deal bundle submitted by the dealer.
-    event DealBundleSubmitted(uint256 dealerIdx, DealBundle bundle);
+    event DealBundleSubmitted(uint32 dealerIdx, DealBundle bundle);
```

Comment:
This is a bit confusing ... why is it uint32? The comment above says it's an address, but it's really just an index right?

Also, uint32 doesn't save gas compared to a uint256 btw

Reply:
I changed the comment. 
So i don't recall exactly why I did the change initially sorry, but I think it's because I don't need uint256 and that uin32 works directly with Rust ( no need to use the special type U256 from ethers).
I'd like to keep it that way now for the sake of avoiding merging this PR for too long but we can definitely revisit with an issue if you feel it.

Reply:
Yeah just leave it, all good